docs/reference.md

```diff
@@ -359,13 +364,30 @@ OpenWhisk has a few system limits, including how much memory an action uses and
 * A user can change the limit when creating the action.
 * A container cannot have more memory allocated than the limit.
 
-### Per namespace #Concurrent Invocation (#) (Default: 100)
+### Per action artifact (MB) (Fixed: 1MB)
+* The maximum size for the artifact (e.g., file name) containing action definition is 1 MB.
+* It's recommended for nodejs action to use a tool to concatenate all source code including dependencies into a single bundled file 
+
+### Per activation payload size (MB) (Fixed: 1MB)
+* Any activations (i.e. invoke action or fire a trigger) the payload can't be larger than 1MB
+* The payload includes parameters for the action or trigger including strings or encoded binary data
+
+### Per namespace concurrent invocation (Default: 100)
 * The number of activations that are currently processed for a namespace cannot exceed 100.
 * The default limit can be statically configured by whisk in consul kvstore.
 * A user is currently not able to change the limits.
 
-
-### Invocations per minute/hour (#) (Fixed: 120/3600)
+### Invocations per minute/hour (Fixed: 120/3600)
 * The rate limit N is set to 120/3600 and limits the number of action invocations in one minute/hour windows.
 * A user cannot change this limit when creating the action.
 * A CLI call that exceeds this limit receives an error code corresponding to TOO_MANY_REQUESTS.
+
+### Per Docker action open files ulimit (Fixed: 64:64)
+* The maximum number of files open is 64 for hard limit and 64 for soft limit
+* The docker run command use the argument `--ulimit nofile=64:64`
+* For more information on the ulimit for open files see the [docker run](https://docs.docker.com/engine/reference/commandline/run) documentation
+
+### Per Docker action number of processes ulimit (Fixed: 512:512)
+* The maximum number of processes available to a user is 512 for hard limit and 512 for soft limit
```

Comment:
The maximum number of processes available to a user is 512 (this applies to both hard and soft limits).